tests/test_forms.py

```diff
@@ -46,6 +46,23 @@ def test_not_required():
     f.clean(['', ''])
 
 
+def test_all_required():
+    f = I18nFormField(widget=I18nTextInput, all_required=True)
+    assert f.clean(['A', 'B', 'C'])
+    with pytest.raises(ValidationError):
+        assert f.clean(['A', '', ''])
```

Comment:
You need to have a separate ``with`` block for *each* of them, as the block will be left after the first exception is raised. Also, you can drop the ``assert`` keyword here.